docs(Guidelines): update guidelines (#767)
PreMiD/Documentation: en/dev/presence/guidelines.md

In some situations, presences may behave unexpectedly or could use some minor changes to improve their functionality. Here is a list of rules that you **must** follow while modifiying presences.

> - If the presence author hasn't made the required changes within a month, you may contact a reviewer to see if you can to modify the presence.

Reviewer: ```suggestion
- If the presence author hasn't been contactable in over a month, you may contact a reviewer to see if you can modify the presence.
```
